File: Tests/VB/TestAllDatabases/TestAllDatabases/BusinessObjects/CustomObjects/AggregateDb/ComputedTestCustom.vb
'
'===============================================================================
'                     EntitySpaces(TM) by EntitySpaces, LLC
'                 A New 2.0 Architecture for the .NET Framework
'                          http://www.entityspaces.net
'===============================================================================
'                       EntitySpaces Version # 2007.0.0730.0
'                       MyGeneration Version # 1.2.0.7
'                           7/30/2007 5:51:12 PM
'-------------------------------------------------------------------------------
'



Imports System.ComponentModel
Imports System.Collections.Generic
Imports System.Text
Imports System.Data

Imports EntitySpaces.Interfaces
Imports EntitySpaces.Core

Namespace BusinessObjects
	Public Partial Class ComputedTest
		Inherits esComputedTest
#Region "Custom Entity Tests"

        Public Function GetDescriptionDoubleQuoteTest() As String
            Return Me.Meta.Columns("DateAdded").Description
        End Function

        Public Function GetDescriptionBackSlashTest() As String
            Return Me.Meta.Columns("SomeDate").Description
        End Function
#End Region
    End Class

End Namespace

File: Tests/CSharp/TestAllDatabases/TestAllDatabases/BusinessObjects/CustomObjects/AggregateDb/ComputedTestCustom.cs
/*
===============================================================================
                     EntitySpaces(TM) by EntitySpaces, LLC
                 A New 2.0 Architecture for the .NET Framework
                          http://www.entityspaces.net
===============================================================================
                       EntitySpaces Version # 2007.0.0730.0
                       MyGeneration Version # 1.2.0.7
                           7/30/2007 5:51:12 PM
-------------------------------------------------------------------------------
*/


using System;
using System.ComponentModel;
using System.Collections.Generic;
using System.Text;
using System.Data;

using EntitySpaces.Interfaces;
using EntitySpaces.Core;

namespace BusinessObjects
{
    public partial class ComputedTest : esComputedTest
    {
        #region Custom Entity Tests

        public string GetDescriptionDoubleQuoteTest()
        {
            return this.Meta.Columns["DateAdded"].Description;
        }

        public string GetDescriptionBackSlashTest()
        {
            return this.Meta.Columns["SomeDate"].Description;
        }
    }

        # endregion
}
